src/pages/404.tsx

```diff
@@ -1,23 +1,13 @@
-import { Stack, Typography } from "@mui/material";
+import { Stack } from "@mui/material";
 import React, { type FC } from "react";
 
 import { text } from "~/assets/text";
+import { PageLayout } from "~/containers/page-layout/index.js";
 
-const styles = {
-  container: {
-    height: "100vh",
-    alignItems: "center",
-    justifyContent: "center",
-  },
-};
+const Custom404: FC = () => (
+  <Stack mt="20vh">
+    <PageLayout title={text.general.pageNotFound} />
+  </Stack>
```

Comment:
Isn't this an anti-pattern now? PageLayout is supposed to be the parent element in pages

Reply:
Since a few routes don't actually use this layout (splash, game, match-making, etc), this gives us the flexibility to use the layout on the pages that are compatible, and maintain the `src/pages` directory as entry point. 

The other option would be to use the layout feature of Next but have some logic to skip it on certain routes, let me know if you think that would be better and I can see about refactoring